src/app/components/Account/AccountLink.tsx

```diff
@@ -17,21 +16,11 @@ export const AccountLink: FC<{
   const { isTablet } = useScreenSize()
   const to = RouteUtils.getAccountRoute(scope, address)
   return (
-    <Typography
-      variant="mono"
-      component="span"
-      sx={
-        plain
-          ? { color: COLORS.grayExtraDark, fontWeight: 400 }
-          : { color: COLORS.brandDark, fontWeight: 700 }
-      }
-    >
+    <Typography variant="mono" component="span">
       {alwaysTrim || isTablet ? (
-        <TrimLinkLabel label={address} to={to} />
-      ) : plain ? (
-        address
+        <TrimLinkLabel label={address} to={to} plain={plain} />
       ) : (
-        <Link component={RouterLink} to={to}>
+        <Link component={RouterLink} to={to} sx={{ fontWeight: plain ? 400 : undefined }}>
```

Comment:
all links should be 700 according to Don

Reply:
> all links should be 700 according to Don (same for TransactionLink). If that is true, why we even need plain?

fixed

> I am a bit confused what plain is doing now (in master not this PR). We render (...)Link component and we use that prop to not render a link?

in most cases it displayed just text instead of a link, I guess so that toggling to ETH addresses would not contain confusing oasis1 addresses :shrug: